extensions/kotlin-serialization/runtime/src/main/kotlin/io/quarkus/kotlin/serialization/JsonConfiguration.kt

```diff
@@ -0,0 +1,110 @@
+package io.quarkus.kotlin.serialization
+
+import io.quarkus.runtime.annotations.ConfigGroup
+import io.quarkus.runtime.annotations.ConfigItem
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.json.JsonNames
+
+@ConfigGroup
+open class JsonConfiguration {
+    /**
+     * Specifies whether default values of Kotlin properties should be encoded.
+     * `false` by default.
+     */
+    @ConfigItem(defaultValue = "false")
+    @JvmField
+    var encodeDefaults: Boolean = true
```

Comment:
Wrong value?

Reply:
hrm.  *ahem*  don't mind the man behind the curtain.